_docs_v5/4_guides/scripting_best_practices.md

```diff
@@ -425,27 +425,17 @@ Once something has failed, and your script has stopped with a pink highlighted l
 
    After a failure, the Script Runner “Pause” button changes to “Retry”. Clicking on the Retry button will re-execute the line the failed. For failures due to timing issues, this will often resolve the issue and allow the script to continue. Make note of the failure and be sure to update your script prior to the next run.
 
-2. Use the Debug Prompt
+1. Use the Debug Prompt
 
    By selecting Script -> Toggle Debug, you can perform arbitrary actions that may be needed to correct the situation without stopping the running script. You can also inspect variables to help determine why something failed.
 
-<!-- 2. Execute Selected Lines While Paused
+1. Execute Selection
 
-   Sometimes re-executing a command or a few other lines of a script can correct problems. This can happen when commanding is over an unreliable transport layer such as UDP or a noisy serial line. For these scenarios, users can highlight the lines of the script they want to run again, right-click, and select “Execute Selected Lines While Paused”. This will run the selected lines again with the full script context (all required variables will still be in scope), and then return. Afterwards you can retry the line that failed or just proceed with “Go”.
+   If only a small section of a script needs to be run, then “Execute Selection" can be used to execute only a small portion of the script. This can also be used when a script is paused or stopped in error.
```

Comment:
We haven't implemented run while paused yet.

Reply:
It sort of works but you get into trouble if you're doing anything that send back to the frontend like a hazardous command. I'll make it disabled when running.